CHANGELOG.md

```diff
@@ -18,9 +18,9 @@
 - [225](https://github.com/vegaprotocol/protos/pull/225) - Added snapshot payload for floating point consensus engine
 - [144](https://github.com/vegaprotocol/protos/pull/144) - Remove validation annotations
 - [228](https://github.com/vegaprotocol/protos/pull/228) - Raw Score and Performance added to Node definition for validator performance reporting
-- [230](https://github.com/vegaprotocol/protos/pull/230) - Implement the transfer commands
```

Comment:
Can you add this back in?

Reply:
Sorry, there were 2 entries for pull 230, I thought the comment had been updated and I had an old entry that needed removing. Restore with this commit https://github.com/vegaprotocol/protos/pull/242/commits/05a820be59fe198d4c9eff3beb42e92d0ec1c697